frontend/src/stores/project.js

```diff
@@ -180,11 +180,17 @@ export const useProjectStore = defineStore("project", {
             return sendRequest("/compiler/compile", this.jsonForCompile, "POST")
                 .then((responseData) => {
                     this.testData.forEach((testCase, index) => {
+                        console.log(responseData[index]);
                         Object.assign(testCase, responseData[index].output);
                     });
                     this.isRunning = true;
                     return true;
                 })
+                .catch((error) => {
+                    this.testData.forEach((testCase) => {
+                        Object.assign(testCase, { error: error.response.data });
```

Comment:
Chyba to można zapisać prościej: `testCase.error = error.response.data`. Chyba, że jest jakiś powód dlaczego tak jest lepiej to daj znać.